src/main/controllers/SignInController.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import {PipRequest} from '../models/request/PipRequest';
+
+export default class SignInController {
+  public get(req: PipRequest, res: Response): void {
+    res.render('sign-in', req.i18n.getDataByLanguage(req.lng)['sign-in']);
+  }
+
+  public post(req: Request, res: Response): void {
+    switch(req.body['sign-in']) {
```

Comment:
Only minor however some of the case statements below have brackets and some don't.

Worth making them consistent (I don't mind which way :) )

Reply:
added brackets